source/fundamentals/auth.txt

```diff
@@ -281,31 +277,20 @@ perform the following:
 #. Supply your credentials using one of the methods in the credential
    provider chain
 
-.. important::
-
-   This method of providing ``MONGODB-AWS`` credentials is available
-   only in the {+driver-long+} v4.8 and later.
-
 To specify the authentication mechanism by using a ``MongoCredential``,
 use the ``MongoCredential.createAwsCredential()`` factory method
 and add the ``MongoCredential`` instance to your ``MongoClient`` as shown
 in the following example:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/MongoDbAwsAuth.java
-   :language: java
-   :dedent:
-   :start-after: start mechOnlyMongoCredential
-   :end-before: end mechOnlyMongoCredential
-   :emphasize-lines: 1,7
+.. literalinclude:: /examples/generated/AuthTest.snippet.aws-credential.kt
+   :language: kotlin
+   :emphasize-lines: 1, 9
 
 To specify the authentication mechanism in the connection string, add
 it as a parameter as shown in the following example:
```

Comment:
Question: why does this sub section not include the two sample auth code snippets in tabs like elsewhere on the page?

Reply:
Unsatisfying answer: we were avoiding reformatting/rewriting unless there was an actual error, and this is the format in the Java doc